Description: Make /etc/runit/2 respect FHS
 On Debian system services directory is /etc/service, not /service.
Author: Dmitry Bogatov <KAction@gnu.org>
Last-Update: 2016-05-31
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/runit-2.1.2/etc/2
+++ b/runit-2.1.2/etc/2
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-PATH=/command:/usr/local/bin:/usr/local/sbin:/bin:/sbin:/usr/bin:/usr/sbin:/usr/X11R6/bin
+PATH=/usr/local/bin:/usr/local/sbin:/bin:/sbin:/usr/bin:/usr/sbin:/usr/X11R6/bin
 
 exec env - PATH=$PATH \
-runsvdir -P /service 'log: ...........................................................................................................................................................................................................................................................................................................................................................................................................'
+runsvdir -P /etc/service 'log: ...........................................................................................................................................................................................................................................................................................................................................................................................................'
